boards/arm/stm32f7/stm32f746g-disco/include/board.h

```diff
@@ -366,7 +374,7 @@
 
 #define GPIO_TP_INT  (GPIO_INPUT|GPIO_FLOAT|GPIO_EXTI|GPIO_PORTI|GPIO_PIN13)
 
-#define FT5x06_I2C_ADDRESS          0x38
+#define FT5x06_I2C_ADDRESS          (0x70 >> 1)
```

Comment:
It is better to list the un shifted address. Is this how it is listed in the datasheet?.

Reply:
I see the address in stm32f746g-disco board sch. https://www.st.com/content/ccc/resource/technical/layouts_and_diagrams/schematic_pack/group1/ff/cd/ce/2d/f8/fb/40/69/mb1191-F746NGH6-C01_schematic/files/mb1191-F746NGH6-C01_schematic.pdf/jcr:content/tranAslations/en.mb1191-F746NGH6-C01_schematic.pdf

![2020-11-26_12-15](https://user-images.githubusercontent.com/1642990/100307574-2d5abb00-2fe1-11eb-8322-a484f64268ef.png)

Reply:
Defining them shifted,  (what you would see on the bus) then shifting them back is confusing.  I would avoid the confusion, and use 7-bit address in the header file and put a comment that says `/* 7 bit address */`

After all what will `i2cdetect` show?  0x38,  If I then grep 0x38. I will not find this file!

Reply:
@ChaochaoCui this is minor change, let's keep it as before.